core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -102,7 +102,7 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
 
     listener.stageIdToData.get((stageId, stageAttemptId)) match {
       case Some(stageData: StageUIData) =>
-        stageData.executorSummary.toSeq.sortBy(_._1).map { case (k, v) =>
+        stageData.executorSummary.toSeq.sortBy(_._1.toLong).map { case (k, v) =>
```

Comment:
I agree @srowen. I can call the sorttable sorting with something like onload. Let me try this.